src/components/searchbar/Searchbar.vue

```diff
@@ -105,9 +107,16 @@ export default {
         v => !!v || 'Bitte Name eintragen',
         v => v.length < 50 || 'Der Name darf nicht länger als 50 Zeichen sein.'
       ],
+      emailRules: [
+        v => (v.length ? /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(v) : true) || 'Bitte eine korrekte Email eintragen'
```

Comment:
Bitte keine eigene Rule. Bei dieser geht z.B. meine E-Mail mit TLD `.name` nicht ...

Reply:
Welche dann? Oder soll ich das prüfen der Email raus lassen?

Reply:
Guck mal hier, das ist ganz schön komplex: https://stackoverflow.com/a/1373724/7140402

Vermutl. reicht prinzipiell die Lösung in der Antwort völlig: https://stackoverflow.com/a/9204568/7140402